wurzel/utils/splitters/sentence_splitter.py

```diff
@@ -12,6 +15,17 @@
 logger = logging.getLogger(__name__)
 
 
+def download_sentence_splitter_model(model_name: str):
+    """Download the SentenceSplitterModel model by name."""
+    # SpaCy models can be installed via pip or via the spacy CLI
+    try:
+        subprocess.check_call([sys.executable, "-m", "spacy", "download", model_name])
```

Comment:
why are we not using 
```suggestion
        spacy.download(model_name)
```
here ?

Reply:
SpaCy does not provide a programmatic endpoint to download the models, only a CLI interface.